plugins/modules/proxmox_kvm.py

```diff
@@ -1053,6 +1065,7 @@ def stop_vm(self, vm, force):
 def main():
     module_args = proxmox_auth_argument_spec()
     kvm_args = dict(
+        archive=dict(type='str'),
```

Comment:
archive can be something like `backup-storage:backup/vm/140/2023-03-08T06:41:23Z` is it a valid path ? I guess the type is used for some typing validation (and check if the value is valid path or not) in such case we should keep the str type ?

Reply:
No, that's not a valid path.

Reply:
i don't know if it's possible to write our own validator or accepting path or valid archive path. We can probably check the value of archive argument and throw and error if it doesn't respect what proxmox except, do you think we should implement this or keep a simple str without validation ?

Reply:
I would simply accept a string. Writing your own validator is somewhat dangerous in case proxmox suddenly extends the syntax, then the code needs to be adjusted.